packages/api/src/object/FetchPageArgs.ts

```diff
@@ -70,7 +70,6 @@ export interface AsyncIterArgs<
   A extends Augments = never,
   S extends NullabilityAdherence = NullabilityAdherence.Default,
 > extends SelectArg<Q, K, R, S>, OrderByArg<Q, PropertyKeys<Q>> {
-  $__EXPERIMENTAL_selectedObjectTypes?: string[];
```

Comment:
Keeping around this type for now in case I want to start adding new things/old things back next week